resources/devices/mobile-phone/sonyericsson.json

```diff
@@ -597,19 +597,6 @@
             "isTablet": "false"
         }
     },
-    "SonyEricsson LT22i": {
```

Comment:
I didn't see any evidence that there was a SonyEricsson version of this in the UA strings, all of them were just "LT22i", and google searches seem to point to the Sony brand rather than SonyEricsson.